examples/nomad-consul-separate-cluster/variables.tf

```diff
@@ -16,13 +15,21 @@
 # ---------------------------------------------------------------------------------------------------------------------
 # OPTIONAL PARAMETERS
 # These parameters have reasonable defaults.
+#
+# aws_profile
+#
```

Comment:
No need to add this to the comment here

Reply:
The intension was to indicate that this parameter aws_profile can be configured by the user and probably should by the user.
What would be a better way to document this?